src/app/app.scss

```diff
@@ -1 +1,11 @@
-@import 'assets/nhsuk-8.1.1.min.css'
+@import 'assets/nhsuk-8.1.1.min.css';
+
+html {
+  font-size: 100%;
+  background-color: #fff;
+}
+
+body {
+  background: #fff;
```

Comment:
We don't need this. The Default background colour is not white. It's `#f0f4f5` which is already applied when importing the css file.

Reply:
I have removed the body section